RELEASE_NOTES

```diff
@@ -3,6 +3,64 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
+v2.7.0 (March 18th 2016)
+------------------------
+
+feature + bugfix release
+- stabilize Cray support
+    - enable 'dynamic' toolchain option by default for Cray* toolchains (#1581)
+    - remove FFTW from the Cray Toolchain Definition (#1585)
+    - add external modules metadata for Cray systems (#1638)
+    - fix independency of Cray toolchains w.r.t. toolchain build environment (#1641, #1647)
+    - remove requirement to use --experimental for Cray toolchains (#1663)
+- enable Python optimization mode in 'eb' (#1357)
+- improved GitHub integration
+    - improve error handling on git commands + better logging for --new-pr/--update-pr (#1590)
+    - use git rather than https in --new-pr/--update-pr (#1602)
+    - add -u as shorthand for --upload-test-report (#1605)
+    - fix --from-pr for PRs that include renamed/deleted files (#1615)
+    - add support for --install-github-token and --check-github (#1616)
+    - fix fetch_easyconfigs_from_pr w.r.t. duplicate files in PRs (#1628)
+- various other enhancements, including:
+    - add support for --search-filename and --terse (REVIEW) (#1577)
+    - support complete bash completion (#1580)
+    - add support for %(*ver)s and %(*shortver)s templates (#1595, #1604)
+        - incl. %(javaver)s, %(javashortver)s, %(perlver)s, %(perlshortver)s, %(pyver)s, %(pyshortver)s, %(rver)s, $(rshortver)s
```

Comment:
no change for you yet, just a comment:
* this seems like the beginning of some templating need that I have in mind since a long time; think "common deps"...
(no idea how this would really grow in real life though, yet)